src/snark/src/relations/arithmetic_programs/qap/tests/test_qap.cpp

```diff
@@ -91,9 +92,13 @@ TEST(relations, qap)
 
     enter_block("Test QAP with binary input");
 
+    test_qap<Fr<alt_bn128_pp> >(1ul << 21, num_inputs, true);
```

Comment:
In the same place ln libsnark, the domain size is taken from a class parameter https://github.com/scipr-lab/libsnark/blob/92a80f74727091fdc40e6021dc42e9f6b67d5176/libsnark/relations/arithmetic_programs/qap/tests/test_qap.cpp#L94
whereas here it is entered manually

Reply:
I assume because here we are choosing the value to match the size of the Zcash circuit? @ebfull did this in his PR; I'm happy to maintain it here.